ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -53,7 +53,7 @@ tests:
       SOURCE_IMAGE_REF: opendatahub-operator
     env:
       IMAGE_REPO: opendatahub-operator
-      RELEASE_VERSION: incubation
+      RELEASE_VERSION: latest
```

Comment:
this will make the incubation with "latest" as version?

Reply:
Right now, we have our latest changes in incubation. So I think we should point latest to it. Or should we point latest to any ODH release ?

Reply:
maybe leave change for "incubation" as-is, and handle them in another PR.